src/main/java/cl/transbank/common/ApiConstants.java

```diff
@@ -26,6 +26,7 @@ public class ApiConstants {
   public static final int EMAIL_LENGTH = 100;
   public static final String TOKEN_TEXT = "token";
   public static final String BUY_ORDER_TEXT = "buyOrder";
+  public static final int REQUEST_TIMEOUT = 30000;
```

Comment:
The timeout must be 10 minutes
```suggestion
  public static final int REQUEST_TIMEOUT = 60000;
```